website/templates/project/addon/node_settings_default.mako

```diff
@@ -34,7 +34,11 @@
                 <p class="break-word">
                     <strong>Current Folder:</strong>
                     <a data-bind="ifnot: folderName() === '', attr.href: urls().files">
-                        {{folderName}}
+                        %if (addon_short_name == 'googledrive'):
+                            {{decodedFolderName}}
+                        %else:
+                            {{folderName}}
+                        %endif
```

Comment:
So the problem with that is that by calling decodedFolderName on every addon creates different behaviors. For example with box, you can name your folder "test test", or "test%20test", which are two different names, but by calling the method will show both as "test test", since %20 is the URL code for " ", while with googleDrive, as far as my understanding goes, since the folder names are saved URL encoded, the names "test test" and "test%20test" are both the same.